lib/trento/activity_log.ex

```diff
@@ -54,10 +62,12 @@ defmodule Trento.ActivityLog do
 
   @spec list_activity_log(map()) ::
           {:ok, list(ActivityLog.t()), Flop.Meta.t()} | {:error, :activity_log_fetch_error}
-  def list_activity_log(params) do
+  def list_activity_log(params, user_has_ability? \\ false) do
```

Comment:
We left on purpose the concept of abilities and "what a user can do with abilities", outside the contexts, please rename this parameter to something else, like `include_all_log_types` or `with_user_management_log_types`.